modules/actions/pool/sync-incentivized-category.ts

```diff
@@ -17,23 +17,15 @@ export const syncIncentivizedCategory = async (chain: Chain) => {
 
     const ids = poolsWithReward.map(({ poolId }) => poolId);
 
-    await prisma.prismaPoolCategory.createMany({
-        data: ids.map((poolId) => ({
-            id: `${chain}-${poolId}-INCENTIVIZED`,
-            category: 'INCENTIVIZED' as const,
-            poolId,
-            chain,
-        })),
-        skipDuplicates: true,
-    });
+    await prisma.$transaction([
+        // Remove incentivized category from pools
+        prisma.$executeRaw`UPDATE "PrismaPool"
+        SET categories = array_remove(categories, 'INCENTIVIZED')
+        WHERE 'INCENTIVIZED' = ANY(categories);`,
```

Comment:
Is this service still needed? seems to be handled in `sync-categories.ts`?

Reply:
We need to keep “incentivised” in sync with the APRs. To avoid repeating the logic, I chained this action with the sync-categories action.